test/val/val_id_test.cpp

```diff
@@ -440,6 +440,45 @@ TEST_F(ValidateIdWithMessage, OpEntryPointReturnTypeBad) {
               HasSubstr("OpEntryPoint Entry Point <id> '1[%1]'s function "
                         "return type is not void."));
 }
+TEST_F(ValidateIdWithMessage, OpEntryPointParameterCountBadInVulkan) {
+  std::string spirv = R"(
+     OpCapability Shader
+     OpMemoryModel Logical GLSL450
+     OpEntryPoint GLCompute %3 ""
```

Comment:
These ids are awkward. Please either re-number the ids in the function so the entry is actually %1 in the source or add an OpName so it's obvious what is being checked.

Reply:
Sounds good, but I took these from the 2 two tests right above which do the same thing. Want me to just clean all 4 of them up while I am at it?

Reply:
SGTM